pages/new-collective-page.js

```diff
@@ -327,14 +313,56 @@ const getCollective = graphql(
             type
           }
         }
-        ...TransactionsAndExpensesFragment
+        # transactions(limit: 3, includeExpenseTransactions: false) {
+        #   ...BudgetItemOrderFragment
+        #   ...BudgetItemExpenseFragment
+        # }
+        # expenses(limit: 3) {
+        #   ...BudgetItemExpenseTypeFragment
+        # }
```

Comment:
Should we un-comment that?

Reply:
Un-commented!

Reply:
(the data was also queried by the Budget section itself, but no need for a weird comment, and this will keep it consistent with current top-level query for the page)